src/ripple/app/tests/Regression_test.cpp

```diff
@@ -95,8 +94,7 @@ struct Regression_test : public beast::unit_test::suite
 
             auto const result = ripple::apply(env.app(),
```

Comment:
Once a function receives `Application&` as a parameter, it makes no sense to also receive `Config const&` as a parameter. You could just call `app.config()`.